tests/errors/binding/main-param.expect

```diff
@@ -0,0 +1,11 @@
+---CODE---
+1
+---STDERR---
+error: `main' requires at least 1 parameters but 0 were provided
+  ┌─ tests/errors/binding/main-param.fil:2:6
+  │
+2 │ comp main[A]<'G:1>() -> () {
+  │      ^^^^ `main' requires at least 1 parameters but 0 were provided
```

Comment:
Can we improve the error message for this case and provide some extra guidance on how to parameterize `main` because that is different from other components

Reply:
This comment was resolved but not actually addressed?

Reply:
I added some more info to the error message in the code? Do you mean to add more than this?

Reply:
I don't see the change. I meant we should say something like: "`main` is a special external component. Parameterizing it correctly requires using the the parameter file/cmdline arguments"